### DISCUSSION TITLE
Airflow containers keep restarting with exit code 137

### DISCUSSION THREAD

**Discussion by copypasteearth** [apache/airflow#37321] [Q&A]
Hi everyone I'm trying to deploy my airflow project on a new Oracle Linux 9 server and when I docker compose up, the containers are constantly restarting with exit code 137. It is working on 2 other servers, one where I didn't do the docker configuration and my dev server which uses docker desktop. I'm thinking docker configuration? What do you think it could be?

**Comment by Swalloow** [comment]
Exit code 137 may imply doesn't have enough memory to finish the work.
You should have to increase memory configuration.
Also check the webserver, scheduler, and worker logs, metrics to find the exact cause.

**Reply by copypasteearth** [reply]
@Swalloow what terminal command can I use to increase memory?
